docs/source/graph_preprocessor.rst

```diff
@@ -0,0 +1,41 @@
+Graph-Based Feature Extraction
+==============================
+
+The GraphFeaturePreprocessor is a scikit-learn compatible preprocessing library that enables feature extraction from graph-structured data.  
```

Comment:
I would avoid saying that the graph preprocessor is a library, because we only have one library now, namely Snap ML. 

I would say something like (please feel free to adapt it as you see fit): "The GraphFeaturePreprocessor is a scikit-learn compatible pre-processor for graph-based data. It provides a set of functions that can enrich raw feature vectors with meaningful graph-related patterns. "